test/integration/ts-dual-esm-cjs-type-module/src/index.ts

```diff
@@ -0,0 +1 @@
+export default () => 'index'
```

Comment:
This fixture can be called ts-dual-package-module, (omit esm-cjs as dual already represents coexistence of both)